src/enet/CMakeLists.txt

```diff
@@ -19,6 +19,9 @@ set(CMAKE_EXTRA_INCLUDE_FILES "sys/types.h" "sys/socket.h")
 check_type_size("socklen_t" HAS_SOCKLEN_T BUILTIN_TYPES_ONLY)
 unset(CMAKE_EXTRA_INCLUDE_FILES)
 
+# C files still around, fetch flags from main configuration
+set(CMAKE_C_FLAGS "${CMAKE_CXX_FLAGS}")
```

Comment:
Now that isn't really the solution I had in mind. Doesn't the C compiler then complain again about C++ flags?

Also, this overwrites all user-set C flags.

Please restore the original snippet, which took into account externally specified flags and the differentiation between C and C++ flags (if the latter is not needed any more, maybe move to just using `COMMON_FLAGS` and then apply them properly).